goss-testing/automated/livecd-preflight-checks

```diff
@@ -27,34 +27,20 @@ set -e
 # This script is run on the PIT node during CSM installs before management
 # nodes are deployed.
 
-# GOSS_BASE should already be set on the PIT node, but just in case we
-# will set it here
-export GOSS_BASE=${GOSS_BASE:-"/opt/cray/tests/install/livecd"}
-
-source "${GOSS_BASE}/automated/run-ncn-tests.sh"
-
-# Validates that a file exists and is a regular file
-function file_exists
-{
-    [[ -e "$1" ]] || err_exit "File does not exist: $1"
-    [[ -f "$1" ]] || err_exit "File exists but is not a regular file: $1"
-}
-
-export GOSS_FILE="${GOSS_BASE}/suites/livecd-preflight-tests.yaml"
-
-file_exists "${GOSS_FILE}"
-
-# The create_tmpvars_file function is defined in run-ncn-tests.sh
-# It creates the temporary variables file and saves the path to it in the ${tmpvars} variable
-create_tmpvars_file || exit 1
+# Many functions and variables used in this script are defined/set/exported
+# in the run-ncn-tests.sh library
 
 echo $'\e[1;33m'Running LiveCD preflight checks \(may take a few minutes to complete\)...$'\e[0m'
 
-# Run the LiveCD preflight Goss test suite
-export GOSS_VARS=${tmpvars}
+# The run-ncn-tests.sh library is located in the same directory as the
+# current script. Using dirname like this could produce a relative path,
+# but that's fine, since we just want to source thie library script.
```

Comment:
minor typo here and in a few files below `thie`

Reply:
Good catch!

Reply:
Fixed!